Makefile

```diff
@@ -121,6 +121,11 @@ golangci-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell dirname $(GOLANGCI_LINT)) v1.63.4 ;\
 	}
 
+.PHONY: yamllint-chart
+yamllint-chart:
+	@files=$$(find testdata/project-v4-with-plugins/dist/chart -name '*.yaml'); \
+    	docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest $$files -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
```

Comment:
Could we add the test without add the linter here?
Or if we add the linter that need to be valid for all yaml files not only dist/chart